Apps/W1/HybridGP/app/src/Migration/Customers/GPCustomerMigrator.codeunit.al

```diff
@@ -173,10 +173,18 @@ codeunit 4018 "GP Customer Migrator"
             CopyStr(MigrationGPCustomer.ADDRESS2, 1, 50), Country, CopyStr(MigrationGPCustomer.ZIPCODE, 1, 20),
             CopyStr(MigrationGPCustomer.CITY, 1, 30));
         CustomerDataMigrationFacade.SetContact(CopyStr(MigrationGPCustomer.CNTCPRSN, 1, 50));
+
+        if (CopyStr(MigrationGPCustomer.PHONE1, 1, 14) = '00000000000000') then
+            MigrationGPCustomer.PHONE1 := '';
+
+        if (CopyStr(MigrationGPCustomer.FAX, 1, 14) = '00000000000000') then
+            MigrationGPCustomer.FAX := '';
```

Comment:
It seems that MigrationGPCustomer is not modified. If we are not updating the value, maybe it is better to assign it to a variable and then use the variable instead. It makes the intent more clear.

Reply:
I have updated this as well to use the new shared function.